.github/workflows/openstack/reboot_watch

```diff
@@ -10,21 +10,43 @@ use POSIX;
 my $RETVAL  = 1;
 my $RETRIES = 0;
 
+# Verify REGEX isn't already in the log before we go into tail mode.  The logodump here is sgnificantly faster than File::Tail.
+# Jump if it's found.
+
+open my $elevate_log_fh, ELEVATE_LOG_PATH;
```

Comment:
FIX: Open the file for reading only and error check if opening the file fails for whatever reason

```
open ( my $elevate_log_fh, '<', ELEVATE_LOG_PATH ) or die "Unable to open ELEVATE_LOG_PATH: $!\n";
```

Reply:
I can add in error checking if it seems beneficial.